src/AlignedSpans.jl

```diff
@@ -40,6 +43,11 @@ end
 const RoundInward = EndpointRoundingMode(RoundUp, RoundDown)
 const RoundEndsDown = EndpointRoundingMode(RoundDown, RoundDown)
```

Comment:
```suggestion
const RoundDown = EndpointRoundingMode(RoundDown, RoundDown)
```

Reply:
I am leaving the decision to rename to you. For me it seems more clear to use rounddown. 
Other files will need that change too.